py/torch_tensorrt/fx/converters/operator.py

```diff
@@ -0,0 +1,1256 @@
+        network,
+        kwargs["input"],
+        prod_value,
+        trt.ElementWiseOperation.SUB,
+        target,
+        name + "_sub",
+    )
+    return sub_value
+
+
+def add_trunc_div(network, target, kwargs, name):
+    return trunc_div(kwargs["input"], kwargs["other"], network, target, name)
+
+
+def add_expand(network, target, kwargs, name):
+    input_t = kwargs["input"]
+    shape = list(kwargs["sizes"])
+
+    input_val = get_trt_tensor(network, input_t, f"{name}_input")
+
+    if network.has_implicit_batch_dimension:
+        shape = shape[1:]
+
+    ranks = len(input_val.shape)
+    # TRT does not support different dimension size
+    assert len(shape) == ranks
+    shape = [input_val.shape[i] if shape[i] == -1 else shape[i] for i in range(ranks)]
+
+    inshape = tuple(input_val.shape)
+    shape = tuple(shape)
+    start = tuple([0] * ranks)
+    stride = tuple(
+        [int(i == o) for i, o in zip(inshape, shape)]
+    )  # stride == 1 if dimensions match, 0 otherwise
+    layer = network.add_slice(input_val, start=start, shape=shape, stride=stride)
+    set_layer_name(layer, target, name)
+    return layer.get_output(0)
+
+
+def add_select(network, target, kwargs, name):
+    input_val = kwargs["input"]
+    if not isinstance(input_val, TRTTensor):
+        raise RuntimeError(
+            f"slice_tensor received input {input_val} that is not part "
+            "of the TensorRT region!"
+        )
+
+    ranks = len(input_val.shape) + (1 if network.has_implicit_batch_dimension else 0)
+    dim = get_positive_dim(cast(int, kwargs["dim"]), ranks)
+    dynamic_shape = has_dynamic_shape(input_val.shape)
+    if network.has_implicit_batch_dimension:
+        if dim == 0:
+            raise RuntimeError(
+                f"We do not support slice_tensor at batch dim when it's implicit, got {dim}!"
+            )
+        dim = dim - 1
+    else:
+        if dynamic_shape:
+            # Check whether slice target dim is dynamic shape dim
+            assert (
+                input_val.shape[dim] != -1
+            ), "Can't select on negative shape dimension!"
+    index = kwargs["index"]
+
+    if index >= input_val.shape[dim]:
+        raise RuntimeError(
+            f"cannot have index greater than the dimension length! {input_val.shape[dim]}"
+        )
+    output_shape = list(input_val.shape)
+    output_shape[dim] = 1
+    if dynamic_shape > 0:
+        output_shape = get_shape_with_dynamic_shape(
+            network, output_shape, input_val, target, name
+        )
+    index_value = torch.tensor(index, dtype=torch.int32)
+    indices_tensor = network.add_constant(
+        index_value.shape, to_numpy(index_value)
+    ).get_output(0)
+    layer = network.add_gather(input_val, indices_tensor, dim)
+    out = layer.get_output(0)
+    if len(out.shape) != 1:
+        layer = network.add_shuffle(out)
+    return layer.get_output(0)
+
+
+def add_slice(network, target, kwargs, name):
```

Comment:
It seems that the `aten` path is now failing in the CI ([as here](https://app.circleci.com/pipelines/github/pytorch/TensorRT/2402/workflows/cd7562f3-3ccf-4ca9-beb4-240a5e93d86f/jobs/11073)). Could be that `aten` includes the `stop` as an endpoint, while `acc` does not? If that is the case, consider adding a keyword arg to the function, like `aten=False`, which determines whether the equation is `output_shape[dim] = (stop_int - start_int) // step_int` or `output_shape[dim] = (stop_int - start_int) // step_int + 1`. Though, I am not sure why the two implementations would differ in the meaning of the endpoint (`stop_int`).